### PR TITLE
Add missing exclusions from phpunit.xml.dist

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -38,6 +38,10 @@
                 <directory>./src/Symfony/Bundle/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/*/Resources</directory>
+                <directory>./src/Symfony/Bridge/*/vendor</directory>
+                <directory>./src/Symfony/Bundle/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/*/vendor</directory>
```
Comment (xabbuh) [inline phpunit.xml.dist]: Not sure if this makes sense. Do you often install dependencies for one of the subpackages only **and** perform code coverage tests then?

Comment (nicolas-grekas) [inline phpunit.xml.dist]: everyday yes

Comment (nicolas-grekas) [inline phpunit.xml.dist]: in fact not exactly what you describe: I sometimes test one single component and forget the vendor there, then later on run the main test suite and it fails becuse phpunit goes into this sub-vendor directory, finds tests to run and boom.

Comment (fabpot) [inline phpunit.xml.dist]: same for me

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**